dbms/src/IO/ReadHelpers.cpp

```diff
@@ -552,9 +552,32 @@ void readCSVStringInto(Vector & s, ReadBuffer & buf, const FormatSettings::CSV &
         while (!buf.eof())
         {
             char * next_pos = buf.position();
-            while (next_pos < buf.buffer().end()
-                && *next_pos != delimiter && *next_pos != '\r' && *next_pos != '\n')    /// NOTE You can make a SIMD version.
-                ++next_pos;
+
+            [&]() {
+#if __SSE2__
+                auto rc = _mm_set1_epi8('\r');
```

Comment:
Shouldn't we include the corresponding `...intrin.h`?

Reply:
Yeah.